### PR TITLE
Add click tracking to best bet boxes and expand all button

### PR DESCRIPTION
**Pull request by kewitham**

### CODE REVIEW

File: _assets/js/utils/gaUtil.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ function gtag() {
     ['.learn-link', 'click', sendGAClickEvent],
     ['.crt-sidenav-subnav-item', 'click', sendGAClickEvent],
     ['.info-box-link', 'click', sendGAClickEvent],
+    ['.best-bet', 'click', sendGAClickEvent],
```
Comment (okkays) [inline _assets/js/utils/gaUtil.js:22]: any way to include info on which best bet they've clicked?

**Review by okkays — CHANGES_REQUESTED** [review]
good idea!

File: _assets/js/utils/gaUtil.js (hunk 1 above)
Comment (kewitham) [reply _assets/js/utils/gaUtil.js:22]: Yes! The event will include the inner text of the link and the link address

**Review by okkays — APPROVED** [review]
very cool, can't wait to see some of this

**Pull request merged**